python/ray/data/tests/test_csv.py

```diff
@@ -796,11 +796,7 @@ def test_csv_read_filter_non_csv_file(ray_start_regular_shared, tmp_path):
     assert ds.to_pandas().equals(df)
 
 
-# NOTE: The last test using the shared ray_start_regular_shared cluster must use the
-# shutdown_only fixture so the shared cluster is shut down, otherwise the below
-# test_write_datasink_ray_remote_args test, which uses a cluster_utils cluster, will
-# fail with a double-init.
-def test_csv_read_no_header(shutdown_only, tmp_path):
+def test_csv_read_no_header(ray_start_regular_shared, tmp_path):
```

Comment:
any ideas why it was not using the shared cluster before? i also looked at the test code but couldn't find any particular reason...

Reply:
Because there was a test (written three years ago) that depended on Ray getting shutdown. But that test isn't in this module anymore, and it calls `ray.shutdown` in the test itself anyway so there's no reason not to use shared clusters at this point